packages/wonder-blocks-dropdown/components/multi-select-menu.js

```diff
@@ -166,6 +166,7 @@ export default class MultiSelectMenu extends React.Component<Props, State> {
                     disabled={item.disabled}
                     key={item.value}
                     label={item.label}
+                    /* eslint-disable-next-line react/jsx-handler-names */
                     onClick={item.onClick}
```

Comment:
This would not have been a violation if we were doing `onClick={this.props.onClick}`.

Reply:
We'd have to pass in the item as well, presumably. Would this be ok: `onClick={() => this.handleClick(item)}`?

Reply:
The menu doesn't do anything with this `onClick`. It's an optional prop on `SelectItem`, to allow the menu creator to attach something like a conversion to a specific item. Maybe we can have something like
```
handleClick(item) { item.onClick(); }
```

Reply:
Unfortunately the lint rule doesn't handle the `onClick={() => this.handleClick(item)}` format so this rule it easy to get around.  It shouldn't be too hard to update it to handle this case as well.